semgrep/semgrep/output.py

```diff
@@ -399,7 +409,13 @@ def build_output(self, color_output: bool) -> str:
         elif output_format == OutputFormat.SARIF:
             return build_sarif_output(self.rule_matches, self.rules)
         elif output_format == OutputFormat.TEXT:
-            return "\n".join(build_normal_output(self.rule_matches, color_output))
+            stats_line = []
+            if sys.stderr.isatty():
```

Comment:
Still not sure why we are passing stats line to output and gating tty instead of outputting to stderr like all other non finding output.

Reply:
Relevant slack convo: https://returntocorp.slack.com/archives/C6A1GN4P3/p1601598463387000?thread_ts=1601596568.384500&cid=C6A1GN4P3

Reply:
It's also less code: call logger.info here: https://github.com/returntocorp/semgrep/blob/develop/semgrep/semgrep/output.py#L346 ?